spec/v2/index.bs

```diff
@@ -803,6 +806,8 @@ The following properties are defined for data type <{Assurance}>:
         <td>String
         <td>M
         <td>The non-empty name of the independent third party engaged to undertake the assurance.
+
+        Advisement: This property is planned to be OPTIONAL in version 3 of these Technical Specifications.
```

Comment:
What do you think about also adding one more explanatory phrase so folks don't get surprised about this change (I've had the situation in the past where SPs are surprised about typo changes without knowing the change was really just to fix a typo). In this case, since it was a typo, think it would be best if we just explain that. Maybe something like "Given this property was incorrectly and unintentionally published in V2 of the Technical Specifications as Mandatory, it will be reverted to Optional in version 3 of the Technical Specifications". What do you think?

Reply:
Agreed